pkg/types/ticket/custom_fields.go

```diff
@@ -265,10 +265,11 @@ func ValidateTicket(ctx context.Context, schema *v2.TicketSchema, ticket *v2.Tic
 	}
 
 	// Look for a matching ticket type
-	foundMatch = false
+	foundMatch = ticket.Type == nil
```

Comment:
I might add a more specific comment here and above to explain why we do this, if a ticket doesn't have a status or a type we don't need to validate, skip the loop in this case